ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -18,12 +18,53 @@ periodics:
     labels:
       master: ci.openshift.redhat.com
     name: azure_build_node_image_centos
-- agent: jenkins
+- agent: kubernetes
   cron: ""
+  decorate: true
   interval: 24h
   labels:
     master: ci.openshift.redhat.com
   name: azure_build_base_image_rhel
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --secret-dir=/usr/local/e2e-azure-secret
+      - --template=/usr/local/build-base-image
+      - --target=build-base-image
+      command:
+      - ci-operator
+      env:
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: release-3.10.yaml
+            name: ci-operator-openshift-openshift-ansible
+      - name: JOB_NAME_SAFE
+        value: build-base-image
+      - name: CLUSTER_TYPE
```

Comment:
Yes, this is a leftover from when we were installing clusters via openshift-ansible.

Reply:
If we enter through the origin-ansible container via the release `ansible.sh` then it would be used.  This is later used as `TYPE` env variable.  I believe @kargakis is correct here.